### PR TITLE
Documentation for setting Azure notebook storage

### PR DESCRIPTION
**Pull request by khalidhuseynov**
### What is this PR for?
This PR adds general info and documentation on setting Azure storage in the `docs/storage.md` folder where we have info about all the supported pluggable storage layers. 

### What type of PR is it?
Documentation

### Todos
* [x] - add docs
* [x] - change description and order in `zeppelin-site.xml.template`

### How should this be tested?
Documentation follows the steps in `zeppelin-site.xml.template`. may need to have account to test.

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by bzz** [comment]
Thank you @khalidhuseynov 👍 

Looks great to me, will merge if there is no more discussion.

File: docs/storage/storage.md [hunk 1]
```diff
@@ -98,3 +99,57 @@ comment the next property:
...
+
+</br>
+#### Notebook Storage  in Azure <a name="Azure"></a>
+
+Using `AzureNotebookRepo` you can connect your Zeppelin with your Azure account for notebook storage.
+
+</br>
+
+First of all, input your `AccountName`, `AccountKey`, and `Share Name` in the file **zeppelin-site.xml** by commenting out and completing the next properties:
+```
```
Comment (AhyoungRyu) [inline docs/storage/storage.md]: @khalidhuseynov Currently this docs is rendered like this
screen shot 2016-05-24 at 3 05 25 pm

So, please add a new line between the each general sentences and code blocks.

**Comment by AhyoungRyu** [comment]
@khalidhuseynov Sorry for my late comment. Except that LGTM 👍

**Comment by khalidhuseynov** [comment]
thanks for review! @AhyoungRyu then do we need to add link [here](https://github.com/apache/incubator-zeppelin/blob/master/docs/_includes/themes/zeppelin/_navigation.html#L87) as well?

**Comment by AhyoungRyu** [comment]
@khalidhuseynov Yeah that would be better :)

**Comment by bzz** [comment]
@khalidhuseynov seems like we need to rebase on latest master and re-trigger CI for this one, before merging.

**Comment by khalidhuseynov** [comment]
@bzz rebased from master

**Pull request closed without merging**